transports/msmq/distributor/configuration.md

```diff
@@ -76,3 +76,87 @@ If self-hosting the endpoint here is the code required to enlist the endpoint wi
 snippet: ConfiguringWorker
 
 Similar to self-hosting, if running NServiceBus prior to Version 6, ensure the `app.config` of the worker contains the `MasterNodeConfig` section to point to the hostname where the distributor process is running.
+
+## Troubleshooting
```

Comment:
Should the troubleshooting section be under configuration? I think it makes sense to be on its own markdown file?

Reply:
Given that this page is quite long already, I'm +1 for a separate troubleshooting page